resources/views/upload.blade.php

```diff
@@ -64,7 +67,7 @@ class="{{ $attributes->get('class') }}"
                       $dispatch('filepond-upload-finished', {'{{ $wireModelAttribute }}': response });
                   }, error, progress);
               },
-              revert: async (filename, load) => {
+              revert: allowRevert ? './revert' : async (filename, load) => {
```

Comment:
What does this `./revert` do?

Reply:
actually it was end point to /revert .

should I put it null instead ?